### PR TITLE
Add optional datum to payToPubKey functions

### PR DESCRIPTION
**Pull request by gege251**
Solves: https://github.com/Liqwid-Labs/plutus-extra/issues/158

### CODE REVIEW

**Review by kozross — CHANGES_REQUESTED** [review]
This needs a minor bump, and a changelog entry.

File: tasty-plutus/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
+## 5.3 -- 2022-01-17
+
+### Changes
```
Comment (kozross) [inline tasty-plutus/CHANGELOG.md:7]: This should be `### Added`

File: tasty-plutus/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+* Added `paysToPubKeyWithDatum`, `paysToWalletWithDatum` including a Datum for pub key outputs and
+  `spendsFromPubKeyWithDatum`, `spendsFromPubKeyWithDatumSigned`, `spendsFromWalletWithDatum`,
+  `spendsFromWalletWithDatumSigned` for pub key inputs
+
 ## 5.2 -- 2022-01-10
 
 ### Changes
```
Comment (kozross) [inline tasty-plutus/CHANGELOG.md:15]: While we're at it, this should be ``### Changed``, not 'Changes'.

**Review by kozross — CHANGES_REQUESTED** [review]

File: tasty-plutus/CHANGELOG.md (hunk 2 above)
Comment (gege251) [reply tasty-plutus/CHANGELOG.md:15]: Sorry, I admit I didn't read the keepachangelog documentation.

**Review by kozross — APPROVED** [review]

**Pull request merged**